### DISCUSSION TITLE
Is it Possible to Use Dataquery to create list of file links?

### DISCUSSION THREAD

**Discussion by dorich** [mdelobelle/metadatamenu#362] [Q&A]
When using the "Link to File" the demo video shows the dataquery box that is used to filter the link possibilities using dataqueryjs. Is it possible to write a NON JS query of the form List FROM "folder name" to filter the results or does the dataquery panel only accept JS input?

Thanks.

**Comment by MoonKraken** [comment]
was wondering this myself as well
